### DISCUSSION TITLE
Who wants to be hired? (May 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#36737] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #35839. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #36738.

**Comment by codewithPoppy** [comment]
Hi, This is Yevhenii and I work as a frontend React developer for over 6 years.
I am looking for full-time remote roles.

Location: Australia
Remote: Yes
Willing to relocate: No
Technologies: React, Next, Remix, Blitz, JS/TS, Mui, ChakraUI, Tailwind, GraphQL, Prisma, Node.js, Rest APIs
Resume/CV: https://drive.google.com/file/d/1hnJ41tPsKtIkdDbc6L-ei4CU8BaZRk8l/view?usp=sharing
Website: https://github.com/codewithPoppy
Email: belovedpoppy0920@gmail.com

**Comment by kellenwiltshire** [comment]
Hi Everyone,

My name is Kellen Wiltshire and I am a passionate full-stack developer specializing in all aspects of Web Development looking for the next step in my journey. I have the experience to create beautiful and functional websites combining the latest frameworks and technologies. I have experience in React(NextJS), Javascript/Typescript, TailwindCSS, Styled Components, as well as traditional HTML5 and CSS3. I am currently look for a new Web Development career opportunity! If you are willing to give me the opportunity I promise you won't find someone more motivated to exceed expectations than me!

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify, Currently learning Vue
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by aboubakarch** [comment]
Hi Everyone,

This is Abou Bakar. I am a passionate frontend developer specializing in various aspects of frontend development and looking for new opportunities. I've more than 3.5 years of experience to build multifunctional web applications for various clients and companies. I've very strong experience in React,NextJS, Javascript/Typescript, TailwindCSS,SCSS,Styled components. I am currently looking for a new career opportunity. I'm very committed to my work and always give my 101% to deliver the best solutions. 
I'm looking forward to hearing anyone of you.
Best Regards.

Location: Lahore, Pakistan
Remote: Yes
Willing to relocate: Yes
Technologies: NextJS, ReactJs, React Native, React Native Web, Redux/Persist-Redux/Redux-ToolKit, TailwindCSS, MaterialUI, AntDesign, SCSS, Strapi, Directus, PostgresSQL, MongoDB, Djangorestframework, Typescript, firebase, Stripe, GCP/Heroku/Netlify/Digital Ocean, GitHub/BitBucket/GitLabs, Jira/Trello/ActiveCollabs, currently learning blockchain.
Resume: [Linkedin](https://www.linkedin.com/in/iamaboubakar/) OR [Resume](https://drive.google.com/file/d/1UMQ-E0ZePPJEFCaPpLREi_6C0z-mH5y_/view?usp=sharing)
Email: [iamaboubakkar@gmail.com](mailto:iamaboubakkar@gmail.com)

**Comment by crackwebai1024** [comment]
Remote : Yes
Willing to relocate: No
working timezone : client's timezone (including UK, EST, CST, PST)
Looking for : fully remote job (full-time)
Resume/CV:
Website: http://mingxing.vercel.app/
Email: piaobristar@gmail.com
mainskills :
        - Frontend
        (SPA - React, Angular, Vue, Svelte)
        (SSR - Next, Nuxt, Sapper)
        (UI - material ui, tailwind CSS, Chakra UI, styled-components, vuetify)
        - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))

**Comment by thesambayo** [comment]
Hi, I'm Samuel and I work as a Frontend Engineer for over 3 years.
I am looking for full-time remote roles.

Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next, Remix, JS/TS, Angular, Tailwind, GraphQL, Node.js, Rest APIs
Resume/CV: https://drive.google.com/file/d/1Vd0Gk1Vc11q4trIXUYeZ_6VkzDKirRr5/view?usp=sharing
Website: [https://www.samueladebayo.tech](https://www.samueladebayo.tech)
Email: [sambayo28@gmail.com](mailto:sambayo28@gmail.com)

**Comment by eric-burel** [comment]
Hi fellow recruiters and entrepreneurs :vulcan_salute: ,

I am a seasoned JavaScript engineer with experiences in building startup products with Next.js, Electron, GraphQL, Meteor...
especially in the field of Big Data visualization and exploration.

My latest work is moving the Vulcan open-source project to a modern "Monorepository".

This architecture is excellent if your company must manage multiple JavaScript applications that may share some common code.

:man_technologist:  Demo here: https://github.com/VulcanJS/vulcan-npm

I can help you moving your company to a similar Monorepo architecture (among other things :)).

My hourly rate: 120$. I am located in France and can work in French or English speaking environments.

You can reach me out via mail: eb@lbke.fr

Looking forward to working with you!

Location: France
Remote: Yes
Willing to relocate: No (I am a freelance)
Technologies: Next.js, Remix, GraphQL, React, Python, Electron
Resume/CV: My publications: https://medium.com/@eric.burel ; my GitHub with OSS projects:  https://github.com/eric-burel and my previous freelance experiences (in French :fr:) https://www.lbke.fr/eric-burel
Website: https://www.lbke.fr
Email: eb@lbke.fr

**Comment by mahdisoultana** [comment]
Hello i'm mahdi from morocco👋
I am a passionate software engineer with over 4 years of development experience. I started my path as a game developer with JavaScript and C# in Untiy 3D engine in 2017 after i started the web development in 2018 HTML ,CSS and in 2019 I fell in love with ReactJS and there the story began i wanted to learn all about it and improve my skills after i found that i need to learn the backend focusing on GraphQL along the journey with React's ecosystem.
Most of my client project was done with Nextjs to ensure that i get the most of SEO benefit for 100%  at google  Speed test this make my client very happy 

Location: Morocco
Remote: YES
Willing to relocate: YES
Technologies: JavasScript ,ReactJS ,,NextJS ,ReactQuery ,GraphQL ,Socket.io,Tailwind ,Express GSAP, Framer-Motion,CSS,Strapi ,MongoDB ,Performance ,Testing.
Resume/CV: https://drive.google.com/file/d/1ZMmRmgBMwipTlOl2sj4Pq8KQpRZ9XapJ/view?usp=sharing
Website: https://github.com/Mahdi-Soultana
Email: [mahdisoultana@gmail.com](mailto:mahdisoultana@gmail.com)

**Comment by CallmeHongmaybe** [comment]
Hi to whom iit may concern, 
I'm Dennis, and before my first job as a Frontend Developer at a mid-size company I was a self taught developer embarking on solo projects with Next.js for nearly 2 years. 
That said I'm very open to any freelancing or part time opportunities. 

Location: VN 
Remote: Yes
Willing to relocate: No 
Technologies: Next.js, Tailwind CSS and Sanity CMS. 
Resume/CV: The portfolio website below should be my CV.  
Website: denniskwan.vercel.app 
Email: quanvihong@gmail.com

**Comment by nmxmxh** [comment]
Location: Abuja, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Redux (with Toolkit & RTK Query), Styled Components, GSAP
Resume/CV: [Resume](https://docs.google.com/document/d/1La-47Ll-3iOiJ_ITJNZmSP0pkOJpfV_TVvcNWybQCEo/edit?usp=sharing)
Website: [nobert.dev](https://nobert.dev/)
Email: okhai0112@gmail.com

**Comment by hasnatsajid** [comment]
Location: Pakistan
Remote: Yes
Willing to relocate: Not sure
Technologies: Next.js, React.js, Node.js, Mongodb, express.js, css,
Resume/CV:  https://files.fm/u/gnbf6js32
Website: https://hasnatsajid.com
Email: hasnatsajid965@gmail.com

**Comment by sabinbaniya** [comment]
Hey there, My name is Sabin. Hope you're doing well.

Location: Nepal
Remote: Yes
Willing to relocate: Not Sure
Technologies: Next.js , React.Js, Tailwind CSS, Material UI, Node.Js, Express.Js, MongoDB
Resume/CV: https://portfolio.sabinbaniya.com.np/assets/sabinbaniya.pdf
Website: https://portfolio.sabinbaniya.com.np/
Email: baniya.sabinn@gmail.com
Twitter: https://twitter.com/sabinbaniya_

**Comment by kumard3** [comment]
I am a front-end developer with 3 years of React Js experience. I've worked on over 20 different projects. I've also contributed to a number of open-source projects.
 
Location: India
Remote: Yes (preferred)
Willing to relocate: yes
Technologies: JavaScript, NodeJS, ReactJS, VueJS, Next.js, MongoDB, Postgres, TailwindCSS, Express
Resume/CV: [resume](https://drive.google.com/file/d/1hikSbE3bWRzWqqtSIrYx8MDFR3T8yB0v/view?usp=sharing)
Website: https://kumard3.me
Email: [kumardeepanshu157@gmail.com](mailto:kumardeepanshu157@gmail.com)

**Comment by Koreankat** [comment]
Hello everyone, My Ali Junior front end developer,
I’m an innovative person , all I’ve built so far are beautifully designed and responsive  web designs.
I’m Looking for a junior role,
Location : Morocco
Remote: Yes
Willing to relocate: Yes
Technologies : Javascript, ReactJS , html , css , node(basic knowledge),git,vscode,express
Resume/CV : https://www.smartr.me/me/ali.jaafari
Website:  https://personal-website-koreankat.vercel.app/
Email: ali007jaafari@gmail.com

**Comment by Anishpras** [comment]
Hello everyone, My name is Anish Prashun. I am a front-end developer and beginner level blockchain developer based in India. I am looking for internships or remote opportunities.

Location: India,
Remote: Yes (Preferable ),
Willing to relocate: Yes,
Technologies: JavaScript/TypeScript, Next.js, React.js, Node.js, MongoDB, SQL, Express.js, Fastify, TailwindCSS, Solidity, Motoko, Rust (Beginner).
Resume/CV: https://www.anishprashun.me/resume.pdf
Website: https://www.anishprashun.me/
Email: anishprashun118@gmail.com

**Comment by idirmosh** [comment]
Somewhat full-stack developer looking for work opportunity, I am somewhat very comfortable with the react eco-system with a passion of learning new things.

**Location**: Greenacres, Florida, US
**Remote**: Yes
**Willing to relocate**: NO
**Technologies**: React. Nextjs. Node. Express. GraphQl, JavaScript. all css-in-js tech. 
**Design**: I have a good eye for user interfaces and web design in general.
**Résumé/CV**: I am currently working on the cv just moved to us not too long ago, currently working on getting the GED.
**Projects**: currently working on an issue-tracker app, the source can be found on my github :https://github.com/idirmosh/issuestracker, its still work on progress, hopefully it would be a demo of my coding skills.
**Website**: idirmosh.dev
**Email**:  idir.mosh@gmail.com
**LinkedIn**: https://www.linkedin.com/in/idirmosh/

Hit me up for any remote work, I would appreciate any opportunity.

**Comment by Sushant5776** [comment]
Hi, I.m Sushant. Being a web developer and an student of B.E. Computer Engineering (currently in final year) at SPPU, Pune. I am experienced in developing softwares in technologies provided below. I am a web3 aspirant and actively looking for jobs in my working domain (Web Developement | Blockchain Developer). You can check my work at https://github.com/Sushant5776 . I would love to work in a collaborative environment along with the experience people in the field. Good luck, have a great time ahead. Thank You! 

**Location:** Pune | Bangalore, India
**Remote:** No
**Willing to relocate:** Yes
**Technologies:** HTML, CSS, TailwindCSS, JavaScript, TypeScript, React.js, Next.js, NoSQL / SQL Databases (Comfortable with Google Firebase, MongoDB), CMS (Sanity, Strapi API), Microservices (FastAPI in Python, Express.js), Blockchain, Solidity (Smart Contracts on Ethereum), C++ (Data Structures and Algorithms).
**Resume/CV:** https://drive.google.com/file/d/1QNZtjDI73m-2PXpE8Rxw6iBusYePXGqY/view?usp=sharing
**Website:** Currently planning to develop!
**Email:** garudkar.sa@gmail.com
